nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -158,36 +183,74 @@ class TfOpIdxMap {
     buffer_map.addDataWhenNotFound(
       empty_buffer, {0, empty_buffer}); /// this represents empty buffer
 
-    auto &variable_map = getIndexMap<const float *, const Tensor *>();
-    auto update_variables = [&variable_map,
-                             &buffer_map](const TfOpNode::Variables &variables,
-                                          bool update_buffer) {
+    auto update_buffer_map = [&buffer_map](const TfOpNode::Variables &variables,
+                                           bool dynamic) {
       for (auto &variable : variables) {
         const float *buf = variable->getData();
-        variable_map.addDataWhenNotFound(buf, variable);
-        if (update_buffer) {
-          buffer_map.addDataWhenNotFound(buf, {variable->bytes(), buf});
-        }
+        assert(buf != nullptr);
```

Comment:
@zhoonit I thought all the tensors are initialized in the first place, which means they've got its buffer. Could you let me know when the user requests a model converting without buffer allocation?

Reply:
NNTrainer requires graph to be compiled/initialized to allocate buffer. If user skip this stage and try to convert, they need to be told with proper error message that they have to compile/initialize the model before requesting convert.

Reply:
Use assert only if it is an internal bug. If it can be triggered by environments or apps, throw exceptions or return error codes. We shouldnt abort user proecess,but allow the app (caller) to handle the error in general.